src/components/Navbar/data.js

```diff
@@ -15,7 +15,7 @@ export const navData = [
     path: "/watchlater",
     icon: <MdWatchLater />,
   },
-  { id: "n5", name: "Playlist", path: "/playlists", icon: <BsPlayBtnFill /> },
+  // { id: "n5", name: "Playlist", path: "/playlists", icon: <BsPlayBtnFill /> },
```

Comment:
remove comments

Reply:
I will need this later when I implement playlist feature :)

Reply:
I will uncomment this when I implement playlist feature,